src/pybind/mgr/dashboard/base.html

```diff
@@ -97,6 +97,25 @@
                 return truncated_float + units[unit];
             };
 
+            rivets.formatters.dimless_binary = function(n){
```

Comment:
Could take the common bits of this and dimless into a function that just takes the 1024/1000 and the list of suffixes?